implementation/jwt-auth/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -710,6 +722,7 @@ Optional<JWTAuthContextInfo> getOptionalContextInfo() {
         contextInfo.setHttpProxyPort(httpProxyPort);
         SmallryeJwtUtils.setContextGroupsPath(contextInfo, groupsPath);
         contextInfo.setExpGracePeriodSecs(expGracePeriodSecs);
+        contextInfo.setClockSkew(mpJwtVerifyClockSkew.orElse(null));
```

Comment:
Please remove a default value for `expGracePeriodSecs` and it is set to something - log that this property is deprecated and that a new one should be used, similarly to how done for other deprecated properties.
The new property should be set to `60` by default to retain the same expectations after  `expGracePeriodSecs` is no longer initialized to 60 by default.
Does it sound reasonable ?

Reply:
@sberyozkin Yes, thank you! Updated